### PR TITLE
Add stay home logos

### PR DESCRIPTION
**Pull request by DilwoarH**
## What

Add Stay at home logos to the landing page banner.

## Why

This is so that we match the branding of the new campaign message.

## Before

Screenshot 2021-01-05 at 09 06 53
Screenshot 2021-01-05 at 09 07 03

## After

Screenshot 2021-01-05 at 09 06 20
Screenshot 2021-01-05 at 09 06 38

### CODE REVIEW

File: app/assets/stylesheets/views/_covid.scss [hunk 1]
```diff
@@ -72,40 +72,6 @@ $c19-landing-page-header-background: #272828;
...
-    padding-top: 0;
-  }
-}
-
-.covid__instruction {
-  @extend %govuk-heading-m;
-  margin-bottom: 0;
-  color: govuk-colour("white");
-}
-
```
Comment (kevindew) [inline app/assets/stylesheets/views/_covid.scss:108]: Nice one cleaning this up 🎉

File: app/assets/stylesheets/views/_covid.scss [hunk 2]
```diff
@@ -313,3 +279,22 @@ $c19-landing-page-header-background: #272828;
...
+  }
+}
+
+.covid__logos {
+  max-width: 100%;
+  height: auto;
+  margin-top: govuk-spacing(4);
+
+  @include govuk-media-query($from: tablet) {
+    max-height: 45px; // the SVG canvas height is too big on IE
```
Comment (kevindew) [inline app/assets/stylesheets/views/_covid.scss:298]: Is this an IE bug or something we can change in the SVG? I notice we specify canvas height of 45 in the SVG already. It'd be useful to have some more info in the commit message about the IE problem, for example source, versions affected.

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<% alt_text = "" %>
+
+<svg class="covid__logos" width="300" height="45" viewBox="0 0 300 45" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="covid-logo__title">
```
Comment (kevindew) [inline app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:3]: As this file only contains SVG markup should it have a file name of _stay-home-logo.svg.erb ?

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  <title id="covid-logo__title"><%= alt_text %></title>
```
Comment (kevindew) [inline app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:4]: Do we need this if the alt_text is empty?

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  </g>
+  <defs>
+    <clipPath id="clip0">
+      <rect width="300" height="45" fill="white"/>
+    </clipPath>
+  </defs>
+  <%= tag.image(src: image_url("stay-home-logo.png"),
+    class: "covid__logos",
+    alt: alt_text,
+    "xlink:href": "") %>
```
Comment (kevindew) [inline app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:47]: What's the purpose of this png? I notice the previous SVG's we have didn't have them.

**Review by kevindew — COMMENTED** [review]
Nice one Dilwoar - screen shots look good. I've got a few questions about implementation.

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 5 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:47]: This is a fall back for browsers that do not support SVG like IE8

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 4 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:4]: We can probably remove this

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 3 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:3]: good shout 👍

File: app/assets/stylesheets/views/_covid.scss (hunk 2 above)
Comment (DilwoarH) [reply app/assets/stylesheets/views/_covid.scss:298]: I believe it affects all IE versions - I will confirm shortly.

Comment (DilwoarH) [reply app/assets/stylesheets/views/_covid.scss:298]: ## IE11

### Without max-height
Screenshot 2021-01-05 at 09 42 57

### With max-height
Screenshot 2021-01-05 at 09 43 10

Comment (DilwoarH) [reply app/assets/stylesheets/views/_covid.scss:298]: Just checked all the other IE versions - same issue

File: app/assets/stylesheets/views/_covid.scss [hunk 6]
```diff
@@ -72,40 +72,6 @@ $c19-landing-page-header-background: #272828;
   }
 }
 
-.covid__instruction-list {
```
Comment (alex-ju) [inline app/assets/stylesheets/views/_covid.scss:75]: Thanks for removing these leftovers after changing the approach in https://github.com/alphagov/collections/pull/2206!

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 5 above)
Comment (kevindew) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb:47]: Do we even support IE8? The page already looks heavily broken there so adding support for IE8 here, while having the  rest of it be broken seems like inconsistent, low value and complex.

It'd be great if we had some guidelines we followed on what browsers we support as it seems rather inconsistent where we (GOV.UK collectively) add fallbacks - seeming to fall to dev preferences than to guidelines. Service manual points to IE11 but this is a long way behind that.

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 3 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb]: Done @kevindew

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 4 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb]: Done

File: app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb (hunk 5 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb]: We don't support IE8 and yes I noticed a lot of the things broken when testing this.

I agree with regarding inconsistency, I will bring this up in our next meeting and maybe put in some guidelines on what should be supported and should be ignored.

SVG support seems to be fairly good so might be worth dropping this:
https://caniuse.com/svg

**Review by kevindew — APPROVED** [review]

**Pull request merged**